frontend/src/app/specification-overview/specification-overview.component.ts

```diff
@@ -38,6 +43,38 @@ export class SpecificationOverviewComponent implements OnInit {
     }
   }
 
+  public async downloadSpecification(specification: Specification, fileType: string) {
+    const latestVersion = specification.versions[0];
+    if (latestVersion !== undefined) {
```

Comment:
If it is unspecified, what happens? And why could this ever happen?

Reply:
I suppose just "defensive programming"

Reply:
Makes sense :) But I'd either expect an error message if it doesn't work, or we just assume that it has to work don't check.

Reply:
Obviously my last comment is lacking in the grammar department.
Correct: 
But I'd either expect an error message if it doesn't work, or we just assume that it has to work _and_ don't check.